pkg/service/labels/constants.go

```diff
@@ -0,0 +1,21 @@
+package labels
+
+const (
+	NameLabel = "Name" //Capital N for Aws
+
+	CreatorLabel = "creator"
+	SpawnerLabel = "spawner"
+	Scope        = "scope"
+
+	ProvisionerLabel         = "provisioner"
+	ClusterNameLabel         = "cluster-name"
+	NodeNameLabel            = "node-name"
+	InstanceLabel            = "instance"
+	NodeLabelSelectorLabel   = "nodeLabelSelector"
+	VpcTagKey                = "vpc"
+	NBTypeTagkey             = "nb-type"
+	NBRegionWkspNetworkStack = "nb-region-ntwk-stk"
```

Comment:
Rather than declaring variables as string, can we declare as
````
type Label struct {
    value string
}

func (l Label) Value() string {
    return TagKey(l.value)
}

ProvisionerLabel = Label{value: "provisioned"}

// usage
// ProvisionerLabel.Value() rather than TagKey(ProvisionerLabel) as we might miss out on adding TagKey in current approach but this approach enfores us.
````

Reply:
refactored this with simple types.